FND/DatesAndTimes/Occurrences.rdf

```diff
@@ -177,24 +184,25 @@ They are modularized this way to minimize the ontological committments that are
 				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label>occurrence based date</rdfs:label>
-		<skos:definition>An OccurrenceBaseDate is a CalculatedDate that is defined with respect to the Occurrence of some OccurrenceKind. The &apos;hasDateValue&apos; property of an OccurrenceBasedDate is not set until the Occurrence happens.  The &apos;triggeredBy&apos; property relates an OccurrenceBasedDate to the OccurrenceKind that gives the meaning of the OccurrenceBasedDate.</skos:definition>
+		<rdfs:label>occurrence-based date</rdfs:label>
+		<skos:definition>calculated date that is defined with respect to the occurrence of some occurrence kind</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>The &apos;hasDateValue&apos; property of an OccurrenceBasedDate is not set until the Occurrence happens.  The &apos;triggeredBy&apos; property relates an OccurrenceBasedDate to the OccurrenceKind that gives the meaning of the OccurrenceBasedDate.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
I could really use an example and more around intended usage. Let's take a scheduled dividend payment date, which is set per a fixed schedule or declared a month or so in advance by the Board.  Would that be suddenly be additionally classified by OccurrenceBasedDate only once the payment happens? 
When is the Occurrence (e.g. a payment) deemed to have "happened"? This is important for the definition of this class. There are various options, taking as an example old fashioned means of payment: when the payment instruction is given, when the check is posted, wehn it arrives, when it is deposited, when the money leaves the sender's account, when it arrives in the receiver's account or when it has cleared in the receiver's account? Where does FIBO have the answer?